### PR TITLE
HTML-19 - Change to capital letters or small letters

### PR DESCRIPTION
**Pull request by burmazovic**
I have wrote the function preslovi().
Function is going thru each character and if it is a lower it is changing it to upper case, and vice versa.

### CODE REVIEW

File: HTML-19/index.html [hunk 1]
```diff
@@ -6,6 +6,14 @@
...
 <body>
-	<script src="oddNumbers.js"></script>
+	<p>
+		Unesi tekst:</p><br>
+		<input id="input" name="text" type="text" value="">
+		<br>
+
+		<button id="submit" type="submit" value="Click me!">
+
+	<p id="ispisPolje"></p>
```
Comment (ban17) [inline HTML-19/index.html]: Mogao bi ovaj indent malo da središ ;)

Comment (ban17) [reply HTML-19/index.html]: Ako već praviš nešto sa submit dugmetom fino bi bilo da sve to okružiš formom a event listener da bude submit na tu formu a ne click na dugme.

File: HTML-19/HTML-19.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                ispis.push(c.toUpperCase());
+            }
+        });
+        ispisPolje.innerText = ispis;
+}
+
+
+
+
+
```
Comment (Iwana279) [inline HTML-19/HTML-19.js]: Ne treba stavljati bespotrebno toliko praznih redova.

File: HTML-19/HTML-19.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+let ispisPolje = document.getElementById("ispisPolje");
```
Comment (ZivojnovicZarko) [inline HTML-19/HTML-19.js]: Please use english for variable and function naming. Also for ids and classes.

File: HTML-19/index.html [hunk 4]
```diff
@@ -6,6 +6,14 @@
 	<title>Odd numbers</title>
 </head>
 <body>
-	<script src="oddNumbers.js"></script>
+	<p>
```
Comment (ZivojnovicZarko) [inline HTML-19/index.html]: Use label instead of `p` tag for text related to input.

File: HTML-19/index.html [hunk 5]
```diff
@@ -6,6 +6,14 @@
...
+		Unesi tekst:</p><br>
+		<input id="input" name="text" type="text" value="">
+		<br>
+
+		<button id="submit" type="submit" value="Click me!">
```
Comment (ZivojnovicZarko) [inline HTML-19/index.html]: This button has no text, it should be 
```
<button id="submit" type="submit">Click me!</button>
```

File: HTML-19/HTML-19.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        let input = document.getElementById("input").value;
+        input.split('').forEach(function(c) {
+            if (c === c.toUpperCase()) {
+                ispis.push(c.toLowerCase());
+            }
+            else  {
+                ispis.push(c.toUpperCase());
+            }
+        });
+        ispisPolje.innerText = ispis;
```
Comment (ZivojnovicZarko) [inline HTML-19/HTML-19.js]: In this case you're going to print an array of the characters, it should be 
`ispisPolje.innerText = ispis.join('');`

**Review by hanii1993 — APPROVED** [review]

**Review by ban17 — APPROVED** [review]

File: HTML-19/index.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Change to capital letters or small letters</title>
+</head>
+<body>
+	<label for="input">Insert words:</label>
```
Comment (ban17) [inline HTML-19/index.html:9]: Form still missing.

**Review by ban17 — CHANGES_REQUESTED** [review]

**Review by ban17 — APPROVED** [review]

File: HTML-19/HTML-19.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+        input.split('').forEach(function(c) {
+            if (c === c.toUpperCase()) {
+                letters.push(c.toLowerCase());
+            }
+            else  {
+                letters.push(c.toUpperCase());
+            }
+        });
+        writeField.innerText = letters.join('');
+}
```
Comment (janko1031) [inline HTML-19/HTML-19.js:18]: there should be one new line at the end of file.

File: HTML-19/index.html [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+	<input id="input" name="text" type="text" value="">
+	<br>
+	<button id="submit" type="submit">Click me!</button>
+	<p id="writeField"></p>
+<script src="HTML-19.js"></script>
+</body>
+</html>
```
Comment (janko1031) [inline HTML-19/index.html:16]: same here, new line should be here.

**Pull request merged**